package.json

```diff
@@ -34,6 +34,7 @@
   "homepage": "https://github.com/egoist/devdocs-desktop#readme",
   "devDependencies": {
     "devtron": "^1.4.0",
+    "debug": "^3.1.0",
```

Comment:
this should be `dependencies` in `app/package.json`?

Reply:
@egoist yep thanks